go.mod

```diff
@@ -4,11 +4,9 @@ go 1.15
 
 require (
 	github.com/bradleyfalzon/ghinstallation v1.1.1
-	github.com/golang/protobuf v1.4.3 // indirect
 	github.com/google/go-github/v32 v32.1.0
 	github.com/kr/text v0.2.0 // indirect
-	github.com/naveensrinivasan/httpcache v1.2.1
-	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
+	github.com/naveensrinivasan/httpcache v1.2.2
```

Comment:
The fix for handling tarball https://github.com/gregjones/httpcache/pull/86.

As the next steps, I will fork this repo into OSSF org.

Reply:
It's fine to keep it in your personal one actually. Any reason to put it in the org?

Reply:
Risk of me blowing off the repository 😉